_data/simple-icons.json

```diff
@@ -1285,6 +1285,11 @@
             "hex": "1EABE2",
             "source": "https://cevo.com/"
         },
+        {
+            "title": "Chainlink",
+            "hex": "2A5ADA",
```

Comment:
On the [brand assets](https://chain.link/brand-assets) page, the primary brand colour is described as `#375BD2`. Would you be able to update it here too?

Reply:
Updated